libs/couch2pg/importer.js

```diff
@@ -0,0 +1,105 @@
+var _ = require('underscore'),
+    Promise = require('rsvp').Promise,
+    format = require('pg-format');
+
+var deleteDocuments = function(db, docIdsToDelete) {
+  if (docIdsToDelete && docIdsToDelete.length > 0) {
+    return db.query(
+      format('DELETE FROM couchdb WHERE doc->>\'_id\' in (%L)',
+        docIdsToDelete));
+  } else {
+    return Promise.resolve();
+  }
+};
+
+var storeSeq = function(db, seq) {
+  return db.query(format('UPDATE couchdb_progress SET seq = %L', seq));
```

Comment:
So that's not what I was talking about, though it's an interesting idea.

What I meant was two things:
 - it makes sense to store the configuration for a postgresdb attaching to a couchdb in that postgresdb to keep that relationship clear and immutable (ie makes it harder to accidentally point a db at the wrong couchdb in the future or whatever)
 - storing configuration in RDBMS' is a bit of a weird one, and tends to get ungainly if every time you need a new configuration variable you add a new column to some table. IMO if you think you're going to store more than 2-3 things you might as well just store a JSON blob (or XML / YAML / EDN whatever) and parse it out.